website/docs/indexes.md

```diff
@@ -0,0 +1,126 @@
+Without an index, each query will need to scan each document in the collection to find the ones that match, which can be resource intensive.
+
+However, if there's an index on the `price` field, the query only scans two documents where their prices are greater than `$900`, instead of all four.
+
+## How to create indexes
+
+Use the `createIndexes()` command to create an index on a collection.
+This command takes two arguments: a document containing the index key (fields to index), and an optional document specifying the index options.
+
+You can create single field indexes or compound indexes.
+
+### Single Field Indexes
+
+Here's an example of creating an index on the `price` field of a products collection:
+
+```js
+db.products.createIndex({ price: 1 })
+```
+
+This creates an ascending index on the `price` field.
+`1` specifies the index direction for ascending order.
+If it's `-1`, it specifies a descending order direction for the index.
+
+### Compound Indexes
+
+For compound indexes, you can create an index key combining multiple fields together as a key.
+Below is an example of a compound index that uses `price` and `category` fields from the `products` collection as the index key.
+
+```js
+db.products.createIndex({ price: 1, category: 1 })
+```
+
+:::note
+
+* If the `createIndex()` command is called for a non-existent collection, it will create the collection and its given indexes.
+* If the `createIndex()` command is called for a non-existent field, an index for the field is created without creating or adding the field to an existing collection.
+* If you attempt to create an index with the same name and key as an existing index, the system will not create a duplicate index.
+Instead, it will simply return the name and key of the existing index, since duplicate indexes would be redundant and inefficient.
+* Meanwhile, any attempt to call `createIndex()` command for an existing index using the same name and different key, _or_ different name but the same key will return an error.
+
+:::
+
+## How to list Indexes
+
+To display a collection's index details, use the `listIndexes()` command.
+
+To return the list of indexes in the `products` collection, use the following command:
+
+```js
+db.runCommand({listIndexes: "products"})
+```
+
+The returned indexes should look like this, showing the default index, single field index, and compound index.
+
+```js
+{
+  cursor: {
+    id: Long("0"),
+    ns: 'db.products',
+    firstBatch: [
+      { v: 2, key: { _id: 1 }, name: '_id_' },
+      { v: 2, key: { price: 1 }, name: 'price_1' },
+      {
+        v: 2,
+        key: { price: 1, category: 1 },
+        name: 'price_1_category_1'
+      }
+    ]
+  },
+  ok: 1
+}
+```
+
+## How to drop indexes
+
+You can also drop all the indexes or a particular index in a specified collection, except the default index (`_id`).
+
+FerretDB supports the use of `dropIndex()` and `dropIndexes()` command.
```

Comment:
```suggestion
FerretDB supports the use of`dropIndexes` command.
```

There is only one command - `dropIndexes`. The rest is either wrappers or handlers of particular clients or dirvers.